src/anemoi/training/config/diagnostics/plot/rollout_eval.yaml

```diff
@@ -63,6 +63,7 @@ callbacks:
   - _target_:  anemoi.training.diagnostics.callbacks.plot.LongRolloutPlots
     rollout:
       - ${dataloader.validation_rollout}
+    video_rollout: ${dataloader.validation_rollout}
```

Comment:
Do we not potentially want the video to rollout to longer than the normal rollout? If I understand correctly you can choose how frequently this is run. I'm imagining that every 3/4 epochs I might want a gif out to 10 days but with the other validation plots maybe I would only want out to 3 days?

Reply:
Yes, you are right. We allow the evaluation on longer rollouts than training rollout.`dataloader.validation_rollout` is an additional variable that has been introduced in the [callback refactor](https://github.com/ecmwf/anemoi-training/pull/60) that enables evaluation on up to `dataloader.validation_rollout` timesteps.

Reply:
what I meant was a `validation_rollout` and a `validation_video_rollout` which was longer. But maybe this isn't something desirable

Reply:
You can set the rollout for validation metrics, plots and the validation video to any number below `dataloader.validation_rollout`. The configuration `dataloader.validation_rollout` makes sure that the validation batches are long enough to perform the validations over longer rollout.

Reply:
Discussed and clarified offline